sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/ResponseStatusCodes.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace OpenTelemetry.Exporter.AzureMonitor
+{
+    internal class ResponseStatusCodes
+    {
+        public const int Success = 200;
+        public const int PartialSuccess = 206;
+        public const int RequestTimeout = 408;
+        public const int ResponseCodeTooManyRequests = 429;
+        public const int ResponseCodeTooManyRequestsOverExtendedTime = 439;
```

Comment:
For 439, I would recommend using the same name as Breeze here "Too many requests and refresh cache". 
But it's still not clear to me how this differs from 429